Misc/NEWS.d/next/Library/2018-07-04-17-14-26.bpo-34044.KWAu4y.rst

```diff
@@ -0,0 +1,2 @@
+subprocess.Popen now copy the startupinfo argument to leave it unchanged:
```

Comment:
Maybe this sounds better?
```
``subprocess.Popen`` now copies the ``startupinfo`` argument to leave it unchanged:
it will modify the copy, so that the same ``STARTUPINFO`` object can be used multiple times.
```

Not sure if core members add a `Patch by` note...

Reply:
I used your version would sounds better.

Nah, I dislike crediting myself. Git can be used for statistics ;-)